message_center_compassion/mappings/field_to_json.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Flückiger Nathan <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class FieldToJson(models.Model):
+
+    _name = "field_to_json"
```

Comment:
```suggestion
    _name = "compassion.field.to.json"
```
Please update everywhere.